apps/acus/pages/api/reports/gameChoicesForPlayerSchedulerReport.ts

```diff
@@ -25,15 +25,15 @@ export default withApiAuthRequired(async (req: NextApiRequest, res: NextApiRespo
         gc.rank as "Choice", 
         g.name as "Game Title",
         g.gm_names as "GM Names",
-        gc.returning_player as "Returning Player",
+        (case when gc.returning_player then 'Yes' else 'No' end) as "Returning Player",
         gs.message as "Message"
       from game_choice gc 
         join game g on gc.game_id = g.id 
-        join slot s on g.slot_id = s.id 
+        join slot s on gc.slot_id = s.id 
         join membership m on gc.member_id = m.id 
         join game_submission gs on m.id = gs.member_id and m.year = gs.year 
         join "user" u on m.user_id = u.id 
-      where gc.year = ${year} 
+      where gc.year = ${year} and gc.game_id in (select g.id from "game" g where g.year = 2023 or (g.id >= 596 and g.id <= 604))
```

Comment:
maybe

```
where gc.year = ${year} and gc.game_id in (select g.id from "game" g where g.year = ${year} or (g.id >= 596 and g.id <= 604))
```

Not a fan of the hardcoded ids there either, but without changing to a flag I don't see a good alternative, so that's good for now.

Reply:
Yeah, that's not even a maybe. That's an artifact of testing in SQLPro for Postgress and copying and pasting it in the code, then testing it. Obviously it works fine, this year.

Nice catch. Thank you! I love code reviews.

Reply:
Yeah, the magic numbers aren't pretty.

If you want a generic one, this one works on ACUS and ACNW:

select * from game where author_id is null

Reply:
that would be better,  one less thing to forget about.

Reply:
I'll do a separate PR for that later